source/Calamari.Testing/CommandTestBuilderContext.cs

```diff
@@ -2,13 +2,17 @@
 using System.Collections.Generic;
 using System.IO;
 using Calamari.Common.Plumbing.Extensions;
+using Calamari.Testing.Tools;
 using Octostache;
 
 namespace Calamari.Testing
 {
     public class CommandTestBuilderContext
     {
         public List<(string? filename, Stream contents)> Files = new List<(string?, Stream)>();
+        
+        public IList<IDeploymentTool> Tools { get; } = (IList<IDeploymentTool>) new List<IDeploymentTool>();
```

Comment:
Exposing was based on the [Sashimi implementation](https://github.com/OctopusDeploy/Sashimi/blob/ccddd736b5047ff6b795c7b9994a51b2c05fd4f6/source/Sashimi.Tests.Shared/Server/TestCalamariCommandBuilder.cs#L49). As far as I can tell there's no real reason for it to be an IList

Reply:
Ah, I see.

The intent there probably was to be flexible in the future in case we want to change this - and with Tools, it probably makes sense to only need simple operations like add/remove which `IList` exposes. But I think it's overkill here since it's just (internal) testing.